src/light/moonlive/catalog_scripts.py

```
@@ -0,0 +1,92 @@
+        "// One array per role: the folder a script lives in is implied by its role and the role by its\n",
+        "// extension, so neither is stored per entry.\n",
+        "#pragma once\n",
+        "#include <cstddef>\n\n",
+        "namespace mm::moonlive {\n\n",
+    ]
+
+    for role, names in by_role.items():
+        lower = role.lower()
+        folder = FOLDER_BY_ROLE[role]
+        parts.append(f"/// Every factory {lower}, by file name. They live in `moonlive/{folder}/`\n")
+        parts.append(f"/// upstream and in the factory script directory on the device.\n")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)